.travis.yml

```diff
@@ -1,20 +1,12 @@
-dist: trusty
-sudo: false
+dist: focal
 language: java
-jdk:
-- oraclejdk8
-install: "./gradlew --version"
-script: ./gradlew --no-daemon -DGRADLE_OPTS="-Xms128m" --stacktrace build -i
-after_success:
-- git status
-- echo $REPOSITORY_USERNAME
-- echo $TRAVIS_PULL_REQUEST
-- echo $TRAVIS_BRANCH
-- if [ master == $TRAVIS_BRANCH -a $TRAVIS_PULL_REQUEST == false -a  ${REPOSITORY_USERNAME:=X}
-  != X ]; then ./gradlew -DGRADLE_OPTS="-Xms128m" release -i; fi
+before_install:
+  - curl -L "http://nusmv.fbk.eu/distrib/NuSMV-2.6.0-zchaff-linux64.tar.gz" | tar --extract --gzip --strip-components=2 -C $HOME/bin "NuSMV-2.6.0-Linux/bin/NuSMV" || true
```

Comment:
> || true

why do you want to ignore failures there?

Reply:
those servers were unstable and the downloads occasionally would fail (and in principle they are cached in `$HOME/bin`)